service/matching/pollerHistory.go

```diff
@@ -91,13 +89,39 @@ func (pollers *pollerHistory) getPollerInfo(earliestAccessTime time.Time) []*typ
 		// TODO add IP, T1396795
 		lastAccessTime := entry.CreateTime()
 		if earliestAccessTime.Before(lastAccessTime) {
+			rps := _defaultTaskDispatchRPS
+			if value.ratePerSecond != nil {
```

Comment:
I don't understand this - why did it get changed?

Reply:
because the default value is overwritten by a user-provided value